doc/server_API.md

```diff
@@ -210,9 +196,9 @@ On success:
 
 
 
-## remove_from_whitelist
+## remove_plate_from_whitelist
 ### Call
-``` DELETE /remove_from_whitelist?id=<WHITELIST_ID>&plate=asgasg/ ```
+``` DELETE /remove_plate_from_whitelist?id=<WHITELIST_UNIQUE_ID>&plate=asgasg/ ```
```

Comment:
`DELETE /remove_plate_from_whitelist?whitelist_name=<WHITELIST_NAME>&plate=<PLATE>/`

Also maybe add different codes for when there's no such whitelist, and there's no such plate on the whitelist? That could be helpful for error handling on Wojciechs side.

Reply:
Also, take a look at this:
https://en.ryte.com/wiki/Trailing_Slashes

I think all REST endpoints should end without trailing slash, as they aren't directories, but I'm open to debate